src/fetcher/base.py

```
@@ -79,6 +94,80 @@ def __init__(
+        wine = os.environ.get("JVLINK_WINE", "wine")
+        registry_targets = [
+            (r"HKLM\Software\JRA-VAN Data Lab.\uid_pass", "servicekey"),
+            (r"HKLM\Software\WOW6432Node\JRA-VAN Data Lab.\uid_pass", "servicekey"),
+            (r"HKCU\Software\JRA-VAN Data Lab.\uid_pass", "servicekey"),
+            (r"HKLM\SOFTWARE\JRA-VAN\JV-Link", "ServiceKey"),
+            (r"HKLM\SOFTWARE\WOW6432Node\JRA-VAN\JV-Link", "ServiceKey"),
+            (r"HKCU\Software\JRA-VAN\JV-Link", "ServiceKey"),
+        ]
+        failures = []
+        for reg_path, value_name in registry_targets:
+            completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)